src/perf/lib/RpsClient.cpp

```diff
@@ -36,6 +36,8 @@ PrintHelp(
         "  -response:<####>            The length of request payloads. (def:%u)\n"
         "  -threads:<####>             The number of threads to use. Defaults and capped to number of cores\n"
         "  -affinitize:<0/1>           Affinitizes threads to a core. (def:0)\n"
+        "  -sendbuf:<0/1>              Whether to use send buffering. (def:0)\n"
+        "  -waitforstart:<0/1>         Whether to wait for start complete event before sending data. (def:0)\n"
```

Comment:
`sendbuf` is fine, but I thought you were just going to separate out `StreamStart` from `StreamSend`. I don't believe you should ever wait for `StreamStart` to complete. But even if we do add support for it, you shouldn't need to change so much. Just call StreamSend in start complete if `waitforstart` is set.

Reply:
> Just call StreamSend in start complete

But that would execute on msquic thread... And when called from System.Net.Quic it would be on user thread

Reply:
I guess, bottom line, I don't think a performance test should do that. So while .NET is currently doing that, I would argue that it should do everything it can not to do that. I don't want to encourage that behavior because it's inherently not performant.